WooCommerce/Classes/ViewRelated/Dashboard/Coupons/MostActiveCouponsCard.swift

```diff
@@ -0,0 +1,254 @@
+                    ServiceLocator.analytics.track(event: .DynamicDashboard.cardRetryTapped(type: .coupons))
+                    Task {
+                        await viewModel.reloadData()
+                    }
+                })
+                .padding(.horizontal, Layout.padding)
+            } else {
+                timeRangeBar
+                    .padding(.horizontal, Layout.padding)
+                    .redacted(reason: viewModel.syncingData ? [.placeholder] : [])
+                    .shimmering(active: viewModel.syncingData)
+
+                Divider()
+
+                couponsList
+
+                viewAllCouponsButton
+                    .padding(.horizontal, Layout.padding)
+                    .redacted(reason: viewModel.syncingData ? [.placeholder] : [])
+                    .shimmering(active: viewModel.syncingData)
+            }
+        }
+        .padding(.vertical, Layout.padding)
+        .background(Color(.listForeground(modal: false)))
+        .clipShape(RoundedRectangle(cornerSize: Layout.cornerSize))
+        .padding(.horizontal, Layout.padding)
+        .sheet(isPresented: $showingCustomRangePicker) {
+            RangedDatePicker(startDate: viewModel.startDateForCustomRange,
+                             endDate: viewModel.endDateForCustomRange,
+                             customApplyButtonTitle: viewModel.buttonTitleForCustomRange,
+                             datesSelected: { start, end in
+                viewModel.didSelectTimeRange(.custom(from: start, to: end))
+            })
+        }
+        .sheet(item: $viewModel.selectedCoupon) { coupon in
+            couponDetailView(coupon: coupon)
+        }
+    }
+}
+
+private extension MostActiveCouponsCard {
+    var header: some View {
+        HStack {
+            Image(systemName: "exclamationmark.circle")
+                .foregroundStyle(Color.secondary)
+                .headlineStyle()
+                .renderedIf(viewModel.syncingError != nil)
+            Text(DashboardCard.CardType.coupons.name)
+                .headlineStyle()
+            Spacer()
+            Menu {
+                Button(Localization.hideCard) {
+                    viewModel.dismiss()
+                }
+            } label: {
+                Image(systemName: "ellipsis")
+                    .foregroundStyle(Color.secondary)
+                    .padding(.leading, Layout.padding)
+                    .padding(.vertical, Layout.hideIconVerticalPadding)
+            }
+            .disabled(viewModel.syncingData)
+        }
+    }
+
+    var couponsList: some View {
+        VStack(alignment: .leading, spacing: Layout.padding) {
+            // Header
+            HStack {
+                Text(Localization.coupons)
+                    .fontWeight(.bold)
+                    .subheadlineStyle()
+                Spacer()
+                Text(Localization.uses)
+                    .fontWeight(.bold)
+                    .subheadlineStyle()
```

Comment:
The labels' fonts look more like semibold to me - bold is a bit too much IMO.

Reply:
Done in d4051370461706e26fb9c403d8f845ef993cefc0